src/neo-vm/ExecutionContext.cs

```diff
@@ -1,14 +1,12 @@
-using System;
-using System.Collections.Generic;
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
 
 namespace Neo.VM
 {
     [DebuggerDisplay("RVCount={RVCount}, InstructionPointer={InstructionPointer}")]
-    public sealed class ExecutionContext
+    public sealed partial class ExecutionContext
```

Comment:
Why `partial`?

Reply:
Because I created a sub-class in a separate file.

Reply:
I means, is not better to create a class instead of sub-class?

Reply:
It's a private class. It is not used by other classes except `ExecutionContext`.